README.md

```diff
@@ -10,20 +10,22 @@ The covenant of the Community Handbook is that if contributors must figure out a
 
 We welcome the Operate First community to contribute to the handbook!
 There are multiple ways to contribute:
-1. Create an issue in our [GitHub repository](https://github.com/operate-first/community-handbook)
-1. Create a pull request (PR) directly against the repository
-1. Label an issue or PR from another part of the project to identify as part of the Community Handbook
+1. Create an issue in our [GitHub repository](https://github.com/operate-first/community-handbook).
+1. Create a pull request (PR) directly against the repository.
+1. Label an issue or PR from another part of the project to identify as part of the Community Handbook.
 
-After creating an issue or PR, assign it to the project ‘Community Handbook’.
-This will cause it to automatically appear in the Community Handbook [project board](https://github.com/orgs/operate-first/projects/20) as a card under the column ‘Proposed content’.
+After creating an issue/PR, please add it to the Community Handbook project.
+You can follow [these directions](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar) to do so.
+After connecting to the project, the issue/PR will appear as a card in the Community Handbook [project board](https://github.com/orgs/operate-first/projects/20) under the column "Proposed content."
 
 Next, we will begin the process of improving the documentation.
-The card moves between columns as the editing and review process progresses; for details on how the review process works, including the criteria to move between columns, refer to the [reviewer guidelines](handbook_reviewer_guidelines.md).
+The card moves between columns as the editing and review process progresses; for details on how the process works, including the criteria to move between columns, refer to the [reviewer guidelines](handbook_reviewer_guidelines.md).
 
 ### Creating an issue to contribute
-To create an issue, select ‘new issue’. If applicable, select the appropriate template:
-* The ‘Missing documentation’ template: if you have discovered a task for which no documentation currently exists in the handbook
-* The ‘Poor documentation’ template: if you have noticed any kind of errors or mistakes in existing documentation, or if you wish for existing documentation to improve in some way
+To create an issue, select "issues" then "new issue." If applicable, select the appropriate template:
```

Comment:
In the TBD markup style guide, I think we want to use this format for "elements of a graphical user interface", with the spelling and capitalization matching whatever we see in the GUI (which helps reader confidence):

"To create an issue, select **Issues**, then choose **New issue**. If applicable ..."